js/utils/loadTemplate.js

```diff
@@ -5,13 +5,7 @@ var path = require('path');
 module.exports = function(templateFile, callback){
 
   var cur_dir = path.dirname(require.main.filename);
-  fs.readFile(cur_dir + path.sep + templateFile, "utf8", function(err, ob){
-
-    if (err) {
-      throw err;
-    }
-    __.templateSettings.variable = "ob";
-    var tmpl = __.template(ob);
-    callback(tmpl);
-  });
```

Comment:
I switched the file read operation to by synchronous as opposed to a-synchronous. My main motivation was that whatever was rendering a view could not reliably know when the rendered content would be in the DOM. We would have to update the view to fire events when the render was complete and that could get cumbersome, particularly with nested views.

The error handling is still there, because JS by default will throw an exception if one happens (where as before the a-sync file read was catching it for you and relied on you explicitly throwing it).